src/components/ChangePlan/PlanCalculator.test.js

```diff
@@ -0,0 +1,374 @@
+import React from 'react';
+import { render, cleanup, waitFor } from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
+import PlanCalculator from './PlanCalculator';
+import { IntlProvider } from 'react-intl';
+import { AppServicesProvider } from '../../services/pure-di';
+import { BrowserRouter } from 'react-router-dom';
+
+describe('PlanCalculator component', () => {
+  afterEach(cleanup);
+
+  const dependencies = {
+    appSessionRef: {
+      current: {
+        userData: {
+          user: {
+            plan: {},
+          },
+        },
+      },
+    },
+  };
+
+  it('should show all plan types in tabs', async () => {
+    // Arrange
+    const planServiceDouble = {
+      getPlanList: () => [],
+      mapCurrentPlanFromTypeOrId: () => {},
+      getPlanTypes: () => ['prepaid', 'subscribers', 'monthly-deliveries'],
+      getBuyUrl: () => '',
+      getPlans: () => [
+        {
+          type: 'prepaid',
+          id: 1,
+          name: '1500-CREDITS',
+          credits: 1500,
+          price: 15,
+          featureSet: 'standard',
+        },
+      ],
+    };
+
+    // Act
+    const { container, getByText } = render(
+      <AppServicesProvider forcedServices={{ ...dependencies, planService: planServiceDouble }}>
+        <IntlProvider onError={() => null}>
```

Comment:
```suggestion
        <DopplerIntlProvider>
```
As we speaked better using this provider as in reports https://github.com/FromDoppler/doppler-webapp/blob/master/src/components/Reports/Reports.test.js#L18